src/auro-dropdown.js

```diff
@@ -215,6 +215,31 @@ class AuroDropdown extends LitElement {
     this.dispatchEvent(event);
   }
 
+  /**
+   * @param {Object} event - Event passed in from the document click event listener that this function gets attached to.
+   * @returns {void} Method used to close the dropdown bib when clicking in the view outside the dropdown.
+   */
+  offClick(event) {
+    // Hide the dropdown content if we clicked anywhere outside auro-dropdown
```

Comment:
I'd recommend not making comments that are conversational. 

```suggestion
    // Dropdown content is hidden when a user clicks outside the element. 
```